includes/handlers/class-webmention-handler.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Class for handling webmention handlers
+*/
+class Webmention_Handler {
+
+	protected $handlers = array();
+
+	/**
+	 * Must be instantiated with at least one handler.
+	 *
+	 * @param array|Webmention_Handler_Base $handler
+	 *
+	 */
+	public function __construct( $handler ) {
```

Comment:
this is a bit confusing! why adding a handler with the constructor? I would prefer to add all bundled handlers by default.